jupyter-extension/src/plugins/mount/mount.tsx

```diff
@@ -508,104 +447,10 @@ export class MountPlugin implements IMountPlugin {
     this._saveInputSpecSignal.emit(this._repoViewInputSpec);
   };
 
-  setShowDatum = async (shouldShow: boolean): Promise<void> => {
```

Comment:
Really nice to clean up this code in favor of the TabPanel, much easier to understand!